src/abilities/Abolished.ts

```diff
@@ -5,28 +5,32 @@ import { Effect } from '../effect';
 import * as arrayUtils from '../utility/arrayUtils';
 import { getPointFacade } from '../utility/pointfacade';
 import { isUndefined } from 'underscore';
+import Game from '../game';
 
 /** Creates the abilities
  * @param {Object} G the game object
  * @return {void}
  */
-export default (G) => {
+export default (G: Game) => {
 	G.abilities[7] = [
 		//	First Ability: Burning Spirit
 		{
 			trigger: 'onOtherDamage',
+
 			require(damage) {
 				if (!this.testRequirements()) {
 					return false;
 				}
-				if (damage === undefined) {
+				// Commented out due to typescript conflicts, but I don't know if that would cause errors, hence why it's not deleted
```

Comment:
Will keep this in mind for eventual deletion. In earlier alpha versions, abilities could get broken doing "NaN" damage in some cases, kinda ruining nice matches, hence why I requested the prototype coder to do something more feasible instead of that.